File: src/main/java/me/aov/sellgui/hooks/PacketEventsHook.java
package me.aov.sellgui.hooks;

import me.aov.sellgui.SellGUIMain;
import me.aov.sellgui.listeners.PacketEventsPacketListener;
import com.github.retrooper.packetevents.PacketEvents;

public class PacketEventsHook {
    public static void register(SellGUIMain main) {
        PacketEvents.getAPI().getEventManager().registerListener(new PacketEventsPacketListener(main));
    }
}

File: src/main/kotlin/me/aov/sellgui/hooks/PacketEventsHook.kt
package me.aov.sellgui.hooks

import com.github.retrooper.packetevents.PacketEvents
import me.aov.sellgui.SellGUIMain
import me.aov.sellgui.listeners.PacketEventsPacketListener

object PacketEventsHook {
    @JvmStatic
    fun register(main: SellGUIMain) {
        PacketEvents.getAPI().eventManager.registerListener(PacketEventsPacketListener(main))
    }
}
